### DISCUSSION TITLE
The documentation is amazing

### DISCUSSION THREAD

**Discussion by xaviduds** [danyspin97/wpaperd#57] [General]
It's so complete, simple and straightforward, inside README.md the application itself is simple and featureful. Thanks for the hard work.

**Comment by danyspin97** [comment]
Many thanks @xaviduds for the compliments!

**Reply by Kemboiray** [reply]
Thank you for this useful app. I think the installation instructions could be improved to include how to install the required dependencies. This would be helpful to noobs like myself. I struggled to build this on **openSUSE Tumbleweed** due to missing libraries (wayland-client and egl). I did a lot of googling to arrive at the solution. Here's how I got the required dependencies:
```sh
$ sudo zypper in wayland-devel 'pkgconfig(egl)'
$ cd /usr/include
$ sudo ln -s wayland/* .
$ cargo install wpaperd
```

**Reply by blatantia** [reply]
THANK YOUUHH @danyspin97 THIS APP IS SO GREATTZZ
